src/theme/components/multiSelect.ts

```diff
@@ -13,6 +7,9 @@ export const multiSelectWidths = {
     minWidth: {
       md: "175px",
     },
+    maxWidth: {
+      md: "200px",
```

Comment:
Yes, this is something for @bigfishdesign13 to view as well. Just remember that we're building this for all teams and 412 is an awkward value. If we can bump it to up to 420 or something more rounded, it might fit better in layouts as well.

Reply:
I missed this initially.  These values are off.  I will add clarifying comments on the other PR.